src/components/ReportWelcomeText.js

```diff
@@ -66,46 +77,66 @@ const ReportWelcomeText = (props) => {
     const isResctrictedRoom = lodashGet(props, 'report.visibility', '') === CONST.REPORT.VISIBILITY.RESTRICTED;
 
     return (
         <Text style={[styles.mt3, styles.mw100, styles.textAlignCenter]}>
-            {!props.shouldIncludeParticipants
-                ? (
-                    <>
-                        <Text>
-                            {isResctrictedRoom
-                                ? `${props.translate('reportActionsView.beginningOfChatHistoryRestrictedPartOne')}`
-                                : `${props.translate('reportActionsView.beginningOfChatHistoryPrivatePartOne')}`}
-                        </Text>
-                        <Text style={[styles.textStrong]}>
-                            {lodashGet(chatUsers, '[0].displayName', '')}
-                        </Text>
-                        <Text>
-                            {isResctrictedRoom
-                                ? `${props.translate('reportActionsView.beginningOfChatHistoryRestrictedPartTwo')}`
-                                : `${props.translate('reportActionsView.beginningOfChatHistoryPrivatePartTwo')}`}
-                        </Text>
-                    </>
-                ) : (
-                    <>
-                        <Text>
-                            {props.translate('reportActionsView.beginningOfChatHistory')}
-                        </Text>
-                        {_.map(chatUsers, ({displayName, pronouns}, index) => (
-                            <Text key={displayName}>
-                                <Text style={[styles.textStrong]}>
-                                    {displayName}
-                                </Text>
-                                {!_.isEmpty(pronouns) && <Text>{` (${pronouns})`}</Text>}
-                                {(index === chatUsers.length - 1) && <Text>.</Text>}
-                                {(index === chatUsers.length - 2) && <Text>{` ${props.translate('common.and')} `}</Text>}
-                                {(index < chatUsers.length - 2) && <Text>, </Text>}
+            {isPolicyExpenseChat && (
+                <>
+                    <Text>
+                        {props.translate('reportActionsView.beginningOfChatHistoryPolicyExpenseChatPartOne')}
+                    </Text>
+                    <Text style={[styles.textStrong]}>
+                        {/* Use the policyExpenseChat owner's first name or their email if it's undefined or an empty string */}
+                        {lodashGet(props.personalDetails, [props.report.ownerEmail, 'firstName']) || props.report.ownerEmail}
+                    </Text>
+                    <Text>
+                        {props.translate('reportActionsView.beginningOfChatHistoryPolicyExpenseChatPartTwo')}
+                    </Text>
+                    <Text style={[styles.textStrong]}>
+                        {lodashGet(props.policies, [`${ONYXKEYS.COLLECTION.POLICY}${props.report.policyID}`, 'name'], 'Unknown Policy')}
```

Comment:
Should we translate `Unknown Policy`?

Reply:
Honestly now I'm thinking we should just make this `''`. We don't use `Policy` in NewDot really and it doesn't really give useful information to the user: https://expensify.slack.com/archives/C21FRDWCV/p1646934419389329?thread_ts=1646933017.374359&cid=C21FRDWCV